lib/ex_aws/rekognition.ex

```diff
@@ -2,37 +2,126 @@ defmodule ExAws.Rekognition do
   @moduledoc """
   Operations on ExAws Rekognition
   """
+  alias S3Object
 
   @actions %{
+    compare_faces: :post,
+    delete_faces: :post,
+    detect_faces: :post,
+    create_collection: :post,
+    delete_collection: :post,
+    describe_collection: :post,
+    list_collections: :post,
     detect_text: :post
   }
 
-  def detect_text(image) when is_binary(image) do
-    image_64 =
-      image
-      |> Base.encode64()
+  @doc """
+  https://docs.aws.amazon.com/rekognition/latest/dg/API_CompareFaces.html
+  """
+  @spec compare_faces(number(), binary() | S3Object.t(), binary() | S3Object.t()) :: %{
+          optional(any) => any
+        }
+  def compare_faces(similarity_threshold \\ 0.8, source_image, target_image)
```

Comment:
Maybe `similarity_threshold \\ 0.8` should be the last parameter of this method, since it have a default value ? Probably source_image can be the first parameter, to be more useful with the pipe operator `|>` ?

Reply:
Yes, makes sense. The arguments follow the order they are presented in the api documentation but we can change for practicality 👍

Reply:
This is the order I followed -> https://docs.aws.amazon.com/rekognition/latest/dg/API_CompareFaces.html